modules/ROOT/pages/working-with-sandboxes.adoc

```diff
@@ -0,0 +1,266 @@
+=  Working With Sandboxes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use Salesforce sandbox organizations to test your portal in a sandbox environment under the same conditions as your production environment. API Experience Hub enables you to manage separate deployment environments for your API portal. You can connect to multiple Salesforce sandbox organizations and switch between them. Manage Salesforce organizations from the *Manage your API portal* page by selecting sandbox environments that are available in the current Salesforce connection.  
+
+== Before You Begin
+Ensure that you have sandbox organizations already set up. For more information about creating sandboxes, see https://help.salesforce.com/s/articleView?id=sf.data_sandbox_create.htm&type=5[Create a Sandbox] in the Salesforce documentation.
+
+== Manage Salesforce Organization Connections
+
+. In the *Manage your API portal* page, click the connection control. 
++
+image::aeh-org-switcher.png["Connection control",15%]
```

Comment:
More context in the alt text? See https://confluence.internal.salesforce.com/display/MTDT/Updating+Alt+Text#UpdatingAltText-alt-text-guidelinesAltTextGuidelinesfromtheStyleReference